msi/release.sh

```diff
@@ -1,34 +1,119 @@
 #!/bin/sh
 
-# Build Git for Windows' .msi files
+# Build Git for Windows .msi file
 
 test -z "$1" && {
-	echo "Usage: $0 <version> [optional components]"
-	exit 1
+    echo
+    echo "Usage: $0 [-v=|--version=|-v ]<version> [(-o=|--outputDir=|-o ) <Output Directory>] [Optional Components]"
+    echo
+    echo "   -v <version>"
+    echo "   -v=<version>"
+    echo "   --version=<version>"
+    echo "       The version of Git for Windows for which to create an installer."
+    echo "       The version should be in the format:"
+    echo
+    echo "           <Major>.<Minor>.<Revision>.<Build/Patch>."
+    echo
+    echo "   -o <Output Directory>"
+    echo "   -o=<Output Directory>"
+    echo "   --outputDir=<OutputDirectory>"
+    echo "       The location the completed installer should be written to."
+    echo "       If this option is not specified, the installer will be written to your"
+    echo "       home directory."
+    echo
+    echo "   [Optional Components]"
+    echo "       A list of additional packages you want bundled with your Git for Windows"
+    echo "       distribution installer."
+    echo
+    echo "       For example, if you want to install emacs with your Git for Windows"
+    echo "       installation, you would run the command:"
+    echo
+    echo "           ./release.sh M.m.r.b mingw-w64-emacs-git"
+    echo
+    echo "       where M.m.r.b is the Major, minor, revision, and build number, "
+    echo "       respectively."
+    echo
```

Comment:
It would look nicer to use `cat <<-EOF`:

```sh
        cat <<-EOF
        
        Usage: $0 [-v=|--version=|-v ]<version> [(-o=|--outputDir=|-o ) <Output Directory>] [Optional Components]
        
           -v <version>
           -v=<version>
           --version=<version>
               The version of Git for Windows for which to create an installer.
               The version should be in the format:
        
[...]
        EOF
```

Also, please keep the indentation to use tabs (which are to be interpreted as 8 spaces).

Reply:
Thanks